lib/ladybug/energy_model/extension.rb

```diff
@@ -50,13 +50,13 @@ def initialize
 
       # Return the absolute path of the measures or nil if there is none, can be used when configuring OSWs
       def measures_dir
-        File.absolute_path(File.join(@root_dir, 'lib/measures/'))
+        File.absolute_path(File.join(@root_dir, 'OpenStudio/measures/'))
```

Comment:
@chriswmackey in the GitHub Repo, the file path is lib/measures/

Reply:
Ok. I'll change this to be realtive instead of absolute since I think we will change the name from lib at some point. Do you have any idea why lib was chosen in the first place?

Reply:
Done. It's using a relative reference now.